### PR TITLE
Johanna/5505 add adnormal flags to bundle

### LINKED ISSUE #5505: Add Abnormal flags to FHIR messages
### Background
California requires some additional fields to be set for them to accept our FHIR messages. One of them is the Abnormal flag. The FHIR mapping is `Observation.interpretation.coding.code`.

### Action requested
Add the abnormal flag to all Observations for results (AoE observations are also Observations but do not need this flag)
Code should be A (display Abnormal) for positive results and N (display Normal) for negative results.

### Acceptance Criteria
Our FHIR messages contain Abnormal flag in the Observation.interpretation for all result Observations (Observations that have a code for a disease and a value for pos/neg/undetermined).

### Additional context
[Example](https://github.com/CDCgov/prime-reportstream/blob/9ce606741e23563b4e4a00e950cd54e1db7667f4/prime-router/src/test/resources/fhirengine/engine/routerDefaults/qual_test_0.fhir#L400-L410)
FHIR value set: https://build.fhir.org/valueset-observation-interpretation.html

Open questions:
- do Undetermined results need an interpretation? what value should it be?

### PR DESCRIPTION
**Pull request by johanna-skylight**
# BACKEND PULL REQUEST

## Related Issue

- Resolves #5505 

## Changes Proposed

- Adds a interpretation to the observation object.
- Creates a codingRecord class to keep the static values related.
- Add an utility method to set the correct values in the coding depending on the test result.
- Updates unit testing.

## Additional Information

None

## Testing

_Tested in dev 7_
- Perform a test(s) with the values positive, negative, inconclusive and verify the bundle has the correct interpretation code.
```
Normal (N) -> Negative, Inconclusive
Abnormal (A) -> Positive
 
```

### CODE REVIEW

File: backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/CodingRecord.java
Comment (johanna-skylight) [inline backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/CodingRecord.java:1]: @BobanL am I overusing this?. I just really like how I can keep related constant values as a group making the implementation easier

**Review by BobanL — DISMISSED** [review]
Changes look good to me!

**Comment by johanna-skylight** [comment]
> Changes look good to me!

sorry! I just removed a variable that I forgot to clean up 😅  I lost the approval

**Review by emyl3 — APPROVED** [review]
LGTM! Thank you, Johanna!

**Review by fzhao99 — APPROVED** [review]
lgtm!

**Pull request merged**